ontotext-yasgui-web-component/src/components/keyboard-shortcuts-dialog/keyboard-shortcuts-dialog.tsx

```diff
@@ -23,25 +31,43 @@ export class KeyboardShortcutsDialog {
   keydownListener(ev: KeyboardEvent) {
     if (ev.key === 'Escape') {
       this.open = false;
+      this.shortcutsOpen.emit(false);
     }
   }
 
+  private groupShortcutsBySection() {
+    this.groupedItems = this.items.reduce((acc, item) => {
+      const section = item.section;
+      if (section) {
+        if (!acc[section]) {
+          acc[section] = [];
+        }
+        acc[section].push(item);
+      }
+      return acc;
+    }, {});
+  }
+
   private openDialog() {
     this.open = true;
+    this.shortcutsOpen.emit(true);
   }
 
   private closeDialog(evt) {
     const target = evt.target as HTMLElement;
     evt.stopPropagation();
     if (target.classList.contains('close-button') || target.classList.contains('dialog-overlay')) {
       this.open = false;
+      this.shortcutsOpen.emit(false);
     }
   }
 
   private buildDialogConfig(): DialogConfig {
     return {
       dialogTitle: this.translationService.translate('yasqe.keyboard_shortcuts.dialog.title'),
-      onClose: this.closeDialog.bind(this)
+      onClose: this.closeDialog.bind(this),
+      position: 'bottom',
+      nonModal: true
```

Comment:
I think it'd be better if this is changed to `isModal`, because `nonModal` sounds weird.